mmv1/products/networkservices/MulticastDomain.yaml

```diff
@@ -0,0 +1,139 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: MulticastDomain
+description: Create a multicast domain in the current project.
+base_url: projects/{{project}}/locations/{{location}}/multicastDomains
+update_mask: true
+self_link:
+  projects/{{project}}/locations/{{location}}/multicastDomains/{{multicast_domain_id}}
+create_url:
+  projects/{{project}}/locations/{{location}}/multicastDomains?multicastDomainId={{multicast_domain_id}}
+update_verb: PATCH
+import_format:
+  - projects/{{project}}/locations/{{location}}/multicastDomains/{{multicast_domain_id}}
+examples:
+  - name: network_services_multicast_domain_basic
+    primary_resource_id: md_test
+    vars:
+      network_name: test-md-network
+      domain_name: test-md-domain
+autogen_async: true
+immutable: true
+async:
+  operation:
+    timeouts:
+      insert_minutes: 20
+      update_minutes: 20
+      delete_minutes: 20
+    base_url: '{{op_id}}'
+  actions:
+    - create
+    - delete
+    - update
+  type: OpAsync
+  result:
+    resource_inside_response: true
+  include_project: false
+autogen_status: TXVsdGljYXN0RG9tYWlu
+parameters:
+  - name: location
+    type: String
+    description: Resource ID segment making up resource `name`. It identifies the
+      resource within its parent collection as described in
+      https://google.aip.dev/122.
+    immutable: true
+    url_param_only: true
+    required: true
+  - name: multicastDomainId
+    type: String
+    description: |-
+      A unique name for the multicast domain.
+      The name is restricted to letters, numbers, and hyphen, with the first
+      character a letter, and the last a letter or a number. The name must not
+      exceed 48 characters.
+    immutable: true
+    url_param_only: true
+    required: true
+properties:
+  - name: adminNetwork
+    type: String
+    description: |-
+      The resource name of the multicast admin VPC network.
+      Use the following format:
+      `projects/{project}/locations/global/networks/{network}`.
+    required: true
+    immutable: true
+  - name: connectionConfig
+    type: NestedObject
+    api_name: connection
+    description: VPC connectivity information.
+    required: true
+    immutable: true
+    properties:
+      - name: connectionType
+        type: String
+        description: |-
+          The VPC connection type.
+          Possible values:
+          NCC
+          SAME_VPC
+        required: true
+        immutable: true
+      - name: nccHub
+        type: String
+        description: |-
```

Comment:
Can you add the note that `When use SAME_VPC, it is required to pass no NCC hub since the domain is only accessible to one network`?